### PR TITLE
Capture, Cancel and Find sales

### PR DESCRIPTION
**Pull request by antoniofilho**
This PR introduces:

* Capture method
* Cancel method
* Find method
* New helper methods to check status code

### CODE REVIEW

File: lib/braspag-rest/payment.rb [hunk 1]
```diff
@@ -17,7 +21,15 @@ class Payment < Hashie::IUTrash
     coerce_key :credit_card, BraspagRest::CreditCard
 
     def authorized?
-      status.to_i == 1
+      status.to_i.eql?(STATUS_AUTHORIZED)
```
Comment (bvicenzo) [inline lib/braspag-rest/payment.rb]: :clap:

**Comment by bvicenzo** [comment]
:cake: :+1:

File: README.md [hunk 2]
```diff
@@ -61,6 +61,28 @@ credit_card.brand = 'Visa'
...
+sale.cancel
+```
+
+### Capture a sale
+
+```rb
+sale = BraspagRest::Sale.find('REQUEST_ID', 'PAYMENT_ID')
+sale.capture
+```
+
```
Comment (pmatiello) [inline README.md]: :clap:

**Comment by pmatiello** [comment]
:tropical_fish:

**Pull request merged**